scenarios/kubernetes_e2e.py

```diff
@@ -186,6 +189,13 @@ def add_files(self, env_files):
         for env_file in env_files:
             self.cmd.extend(['--env-file', test_infra(env_file)])
 
+    def add_k8s(self, k8s, *repos):
+        for repo in repos:
+            self.cmd.extend([
+                '-v', '%s/%s:/go/src/k8s.io/%s' % (k8s, repo, repo)])
+        self.cmd.extend(['-v', '%s/release:/go/src/k8s.io/release' % k8s])
+        # TODO(fejta): use bazel and remove this
+        self.cmd.extend(['-v', '/var/run/docker.sock:/var/run/docker.sock'])
```

Comment:
we have `--docker-in-docker` flag doing the same thing, seems anyone set `--build` should set that flag.